### ISSUE TITLE
Keeping clipboard Information

### ISSUE THREAD

**Issue by LethaboNt** [Wox-launcher/Wox.Plugin.ClipboardManager#6]
Is there a way to keep clipboard information after restarting pc?

**Comment by lances101** [comment]
Nope. The plugin stores the data in memory only. Persistence would be a good feature though.

**Comment by LethaboNt** [comment]
Thank you @lances101. Yes, it would be an awesome feature.

**Comment by lances101** [comment]
I'll keep it open so it still appears as a suggestion.

**Comment by linuxiamo** [comment]
Also the opposite would be good: delete the clipboard entirely, or delete single entries.
